datadog/resource_datadog_integration_gcp.go

```diff
@@ -16,11 +16,12 @@ var integrationGcpMutex = sync.Mutex{}
 
 func resourceDatadogIntegrationGcp() *schema.Resource {
 	return &schema.Resource{
-		Description:   "Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.",
-		CreateContext: resourceDatadogIntegrationGcpCreate,
-		ReadContext:   resourceDatadogIntegrationGcpRead,
-		UpdateContext: resourceDatadogIntegrationGcpUpdate,
-		DeleteContext: resourceDatadogIntegrationGcpDelete,
+		Description:        "This resource is deprecated — use the gcp_sts resource instead. Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.",
```

Comment:
```suggestion
		Description:        "This resource is deprecated — use the datadog_integration_gcp_sts resource instead. Provides a Datadog - Google Cloud Platform integration resource. This can be used to create and manage Datadog - Google Cloud Platform integration.",
```